examples/local_evaluation/run_local_evaluation.py

```diff
@@ -1,26 +1,39 @@
 from pathlib import Path
 
-from eval_framework.llm.models import Llama31_8B_Instruct_API
+from eval_framework.llm.huggingface_llm import HFLLM
 from eval_framework.main import main
+from eval_framework.task_loader import load_extra_tasks
 from eval_framework.tasks.eval_config import EvalConfig
+from template_formatting.formatter import HFFormatter
+
+
+# Define your model if needed
+class MyHuggingFaceModel(HFLLM):
+    LLM_NAME = "microsoft/DialoGPT-medium"
+    DEFAULT_FORMATTER = HFFormatter("microsoft/DialoGPT-medium")
 
-# from eval_framework.task_loader import load_extra_tasks
 
 if __name__ == "__main__":
+    # Initialize your model
+    llm = MyHuggingFaceModel()
+
     # Using Llama 3.1 8B Instruct hosted on Aleph Alpha Research API
     # This requires an AA_TOKEN to be set in the .env file
-    llm = Llama31_8B_Instruct_API()
+    # llm = Llama31_8B_Instruct_API()
 
     # Load custom tasks from the specified directory or file:
-    # load_extra_tasks(["examples/custom_tasks/"])
+    load_extra_tasks(
+        ["/home/gregory.schott/repos/eval-framework-companion-bis/src/eval_framework_companion/tasks/benchmarks"]
+    )
```

Comment:
For the example, we should probably keep the generic `examples/custom_tasks` folder